glossary.asciidoc

```diff
@@ -470,15 +465,15 @@ soft fork::
 
 SPHINX Mix Format::
     A particular technique for Onion Routing used in the Lightning Network and invented by George Danezis and Ian Goldberg in 2009.
-    With the SPHINX Mix Format, each message of the onion package is padded with some random data so that no single hop can estimate how far on the route they are.
+    With the SPHINX Mix Format, each message of the onion package is padded with some random data so that no single hop can estimate how far along the route it has traveled.
     While the privacy of the sender and receiver of the payment is protected, each node is still able to return an error message along the path to the originator of the message.
     The paper can be found at https://cypherpunks.ca/~iang/pubs/Sphinx_Oakland09.pdf
 
-Submarine Swaps::
-    Submarine Swaps enable transfers between on-chain addresses and off-chain locations, like the Lightning Network. Just as standard LN transfers chain payments by means of HTLCs that make the final claim on funds conditional on the recipient revealing a secret to all links in the chain, Submarine Swaps use the same logic and procedure to transfer funds across the on-chain/off-chain barrier with minimal trust. They can also be used to enable transfers from another chain, say Litecoin, to an off-chain LN address. Reverse Submarine Swaps allow bitcoin transfers in the opposite direction that is, from an off-chain LN location to an on-chain address.
+Submarine Swap::
+    A Submarine Swaps enables transfers between on-chain Bitcoin addresses and off-chain locations, like the Lightning Network. Just as standard LN transfers chain payments by means of HTLCs that make the final claim on funds conditional on the recipient revealing a secret to all links in the chain, Submarine Swaps use the same logic and procedure to transfer funds across the on-chain/off-chain barrier with minimal trust. They can also be used to enable transfers from another chain, say Litecoin, to an off-chain LN address. Reverse Submarine Swaps allow bitcoin transfers in the opposite direction, from an off-chain LN location to an on-chain address.
 
-timelocks::
-    A timelock is a type of encumbrance that restricts the spending of some bitcoin until a specified future time or block height. Timelocks feature prominently in many Bitcoin contracts, including payment channels and hashed timelock contracts.
+timelock::
+    A timelock is a type of encumbrance that restricts the spending of some bitcoin until a specified future time or block height. Timelocks feature prominently in many Bitcoin contracts, including payment channels and Hashed Timelock Contracts.
```

Comment:
I believe it should be "Hash Time Locked Contracts"

Reply:
If you look earlier in the Glossary under HTLC you will find: "A Hashed TimeLock Contract or HTLC is a class of payments that use hashlocks and timelocks"
Also, see the link below which uses "Hashed TimeLock Contract" - note the camelcase capital "L" in TimeLock.
 https://academy.binance.com/en/glossary/hashed-timelock-contract

Reply:
Oh, I don't know then. As a reference for HTLC's definition i was using the routing.asciidoc chapter, but looking a bit on the web it seems like the two names are used interchangeably, so I'm really not sure about this one

Reply:
I think we will just aim to be consistent in final copyediting